tools/admin-edit/index.html

```diff
@@ -42,6 +43,20 @@ <h1>Admin Edit</h1>
       </form>
     </div>
 
+    <div>
+      <label for="headers-container">Headers</label>
+      <div id="headers-container">
+        <div id="auth-token-header" class="header">
+          <input type="text" class="header-name" placeholder="Header name" required="" value="x-auth-token">
+          <input type="text" class="header-value" placeholder="Header value" required="">
+          <button type="button" class="remove-header" aria-label="Delete header">
+            <span class="icon icon-trash-delete-bin"></span>
+          </button>
+        </div>
+      </div>
+      <button type="button" id="add-header" class="button">Add Header</button>
+    </div>
```

Comment:
are these part of the form submission? shouldn't they be inside the form, not after it (and the submit button)?

Reply:
They are part of the form submission and get appended to the headers. There are 2 forms on this page - admin form and the body form, and since the headers get sent on submission on wither of them, associating them with one particular form did not seem correct so I kept them out of both.